is_core/utils/__init__.py

```diff
@@ -146,10 +176,11 @@ def get_cls_or_inst_method_or_property_data(field_name, cls_or_inst, fun_kwargs)
 def get_cls_or_inst_readonly_data(field_name, cls_or_inst, fun_kwargs):
     if '__' in field_name:
         current_field_name, next_field_name = field_name.split('__', 1)
-        next_cls_or_inst = getattr(cls_or_inst, current_field_name)
         field = get_field_from_cls_or_inst_or_none(cls_or_inst, current_field_name)
-        if field and hasattr(field, 'rel'):
-            next_cls_or_inst = next_cls_or_inst or field.rel.to
+        next_cls_or_inst = (
+            field.rel.to if field and hasattr(field, 'rel') and not hasattr(cls_or_inst, current_field_name)
+            else getattr(cls_or_inst, current_field_name)
+        )
```

Comment:
tady byl bug v určitých případech to pro django 1.9 nefungovalo. Pokud se zavolá getattr na onetoone tak v 1.9 vrací attrerror a to je přesně to co nechceme